elastic/kibana
# [Infra ML] Add feedback button: Anomaly detection for hosts and pods

Closes #175792

## Summary

This PR adds a feedback button to the ML flyout in Infra.

<img width="1834" alt="image" src="https://github.com/elastic/kibana/assets/14139027/296ea04d-38d0-41bd-9b9b-28da28d3d403">


## Small UI fixes

This PR makes the tabs inside the flyout consistent with other UI flyouts ( smaller size and aligns them with the title and places them in the header)

| Before | After |
| ------- | -------------- |
| ![image](https://github.com/elastic/kibana/assets/14139027/80a66498-56bf-44f9-af0d-50c2423498f3) | ![image](https://github.com/elastic/kibana/assets/14139027/300d2281-71a6-421e-aef9-5f3c57d668b4) |

## Testing 
- Go to Inventory and click on the Anomaly Detection (Top menu, next to Settings)
   - Feedback button should be visible
   - Click on the feedback button
      - The first step (without any jobs enabled) should not prefill the first question (click on `Enable` for host/pod job)
      - The second step (host or pod) should prefill the first question with the host or pod answer
      - Same on 3rd step (follow the URL at the bottom of the screen to check the params):
     
       https://github.com/elastic/kibana/assets/14139027/8b583a4b-aefa-4694-ab17-98066df97b6b

      - (note that it shouldn't prefill the first question if both pod and host jobs are enabled)
       ![image](https://github.com/elastic/kibana/assets/14139027/d3e0a1b3-02ea-44e1-9ee8-18f1ce216c4f)
